geomesa-accumulo/geomesa-accumulo-datastore/src/test/resources/log4j.xml

```diff
@@ -22,9 +22,6 @@
     <category name="org.apache.curator">
         <priority value="error"/>
     </category>
-    <category name="org.apache.curator">
```

Comment:
Seeing the above block, one of them is unnecessary.  Are any of the 'warn' level log messages worth anything?

Reply:
generally i'd say yes, but the mini cluster tends to spam a lot of warnings so we moved everything to error in that pr. this was just to fix the oversight of leaving in the duplicate entry.